examples/lighting-app/nrfconnect/prj.conf

```diff
@@ -33,3 +33,7 @@ CONFIG_BT_DEVICE_NAME="ChipLight"
 # Additional configs for debbugging experience.
 CONFIG_THREAD_NAME=y
 CONFIG_MPU_STACK_GUARD=y
+
+# CHIP configuration
+CONFIG_CHIP_PROJECT_CONFIG="main/include/CHIPProjectConfig.h"
```

Comment:
Could we set those strings as a default value for this option, to not have to paste it under many examples? I guess that we cannot move it to the samples-default.conf, because pigweed-app doesn't use OpenThread and CHIP.

Reply:
The thing is the same Kconfig file will also be used for CHIP in NCS, so I'm not sure we want to set the default which is unlikely to be valid for applications other than CHIP examples (most apps probably don't conform to the structure where project headers are put in main/include/).